lib/link_expansion/content_cache.rb

```diff
@@ -40,10 +41,25 @@ def editions(content_ids)
       locale_fallback_order: locale_fallback_order,
       state_fallback_order: state_fallback_order,
     )
-    Edition.with_document.includes(:document).where(id: edition_ids).all
+    Edition
+      .joins(
+        <<-SQL.strip_heredoc
+          LEFT OUTER JOIN unpublishings
+          ON unpublishings.edition_id = editions.id
+          AND editions.state = 'unpublished'
+        SQL
+      )
+      .with_document
+      .includes(:document)
```

Comment:
I think this might be redundant with a pluck? the with_document should add the join.